### PR TITLE
Matrix conversion

### PR DESCRIPTION
**Pull request by parthshahva**
This PR is for #195. Updates to file conversion and functional tests for new implementation

### CODE REVIEW

File: tests/functional/validation.py [hunk 1]
```diff
@@ -15,7 +17,17 @@
...
+deployment_stage = os.getenv('DEPLOYMENT_STAGE')
+dss_config = HCAConfig()
+dss_config['DSSClient'] = {}
+if deployment_stage == "prod" or deployment_stage == "dev":
+    dss_config['DSSClient']['swagger_url'] = "https://dss.data.humancellatlas.org/v1/swagger.json"
+elif deployment_stage == "staging":
+    dss_config['DSSClient']['swagger_url'] = f"https://dss.staging.data.humancellatlas.org/v1/swagger.json"
+else:
+    dss_config['DSSClient']['swagger_url'] = f"https://dss.integration.data.humancellatlas.org/v1/swagger.json"
+DSS_CLIENT = hca.dss.DSSClient(config=dss_config)
```
Comment (calvinnhieu) [inline tests/functional/validation.py]: Will consolidate this function in separate PR

**Comment by calvinnhieu** [comment]
Hm why didn't codecov run?

**Comment by calvinnhieu** [comment]
Just a thought, do we want to separate the results S3 bucket into one for Redshift query results and one for expression matrices? We probably want to clean up the former with an expiration policy and not the latter

**Comment by parthshahva** [comment]
I have no idea why neither Travis nor Codecov ran 🤔

**Comment by parthshahva** [comment]
https://github.com/HumanCellAtlas/matrix-service/issues/236 created

**Pull request merged**